src/interface/variable_pack.hpp

```diff
@@ -154,7 +154,7 @@ void FillVarView(const vpack_types::VarList<T> &vars, PackIndexMap *vmap,
   auto host_sp = Kokkos::create_mirror_view(Kokkos::HostSpace(), sparse_assoc);
 
   int vindex = 0;
-  int sparse_start;
+  int sparse_start = -22000; // Initialize to an obviously wrong number
```

Comment:
We should perhaps formalize this throughout the code and make a habit of initializing to [signal NaNs](https://en.cppreference.com/w/cpp/types/numeric_limits/signaling_NaN). That's for another time, though.

Reply:
Should we create an issue?

Reply:
yes good idea